app/templates/ui/app/create/create.controller.js

```diff
@@ -44,20 +81,42 @@
     });
 
     function submit() {
-      mlRest.createDocument(ctrl.person, {
-        format: 'json',
-        directory: '/content/',
-        extension: '.json',
-        collection: ['data', 'data/people']
-        // TODO: add read/update permissions here like this:
-        // 'perm:sample-role': 'read',
-        // 'perm:sample-role': 'update'
-      }).then(function(response) {
-        toast.success('Created');
-        $state.go('root.view', { uri: response.replace(/(.*\?uri=)/, '') });
-      }, function(response) {
-        toast.danger(response.data);
-      });
+      var extension = '.json';
+      var data = ctrl.person;
+      if (ctrl.person.docFormat == 'xml') {
+        extension = '.xml';
+        var wrap = {
+          xml : ctrl.person
+        };
+        data = x2js.json2xml_str(wrap);
```

Comment:
Can you check if gulp build works on your code? You might need to disable/enable jshint camelcase around this line. See detail.ctrl how to do that..